Sources/NetworkProtection/WireGuardKit/Endpoint.swift

```diff
@@ -80,8 +89,8 @@ extension Endpoint {
     }
 }
 
-extension NWEndpoint.Host: CustomStringConvertible {
-    public var description: String {
+extension NWEndpoint.Host {
+    public var hostWithoutPort: String {
```

Comment:
I changed this just to prove to myself that no other part of the codebase was using this implicitly, and relying on the behaviour. Now it's very clear that you're getting the host without the port.